src/ds/d500/d500-fw-update-device.cpp

```diff
@@ -38,6 +38,13 @@ ds_d500_update_device::ds_d500_update_device( std::shared_ptr< const device_info
     void ds_d500_update_device::update(const void* fw_image, int fw_image_size, rs2_update_progress_callback_sptr update_progress_callback) const
     {
         update_device::update( fw_image, fw_image_size, update_progress_callback );
+
+        if (_wait_instead_of_sampling_manifest_reset)  // when dfu monitoring is not enabled by FW
```

Comment:
rename to `_is_dfu_monitoring_supported`?

Reply:
ok